main.go

```diff
@@ -380,6 +382,40 @@ var idCmd = &cobra.Command{
 	},
 }
 
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "shows current version of bidbot",
+	Args:  cobra.ExactArgs(0),
+	Run: func(c *cobra.Command, args []string) {
+		local := buildinfo.Summary()
+		fail := func(format string, vars ...interface{}) {
+			fmt.Printf("local%s\n", local)
+			log.Fatalf(format, vars...)
+		}
+		res, err := http.Get(urlFor("version"))
+		if err != nil {
+			fail(err.Error())
+		}
+		defer func() {
+			err := res.Body.Close()
+			if err != nil {
+				fail(err.Error())
+			}
+		}()
+		b, _ := ioutil.ReadAll(res.Body)
+		if res.StatusCode != http.StatusOK {
+			fail("%s: %s", res.Status, string(b))
+		}
+		remote := string(b)
+		if remote == local {
+			fmt.Println(local)
+			return
+		}
+		fmt.Printf("local%s\n", local)
+		fmt.Printf("\ndaemon%s\n", remote)
```

Comment:
I'd add some:
`fmt.Println("WARNING! You local and remote version don't match:")`

So shouting a bit more to prevent a situation in which they forgot to restart `bidbot daemon`.